packages/lib-classifier/src/plugins/tasks/SurveyTask/models/SurveyAnnotation.spec.js

```diff
@@ -108,6 +112,33 @@ describe('Model > SurveyAnnotation', function () {
     })
   })
 
+  describe('with _choiceInProgress as true', function () {
+    let surveyAnnotationInProgress
+
+    before(function () {
+      surveyAnnotationInProgress = SurveyAnnotation.create({
+        id: 'survey1',
+        task: 'T0',
+        taskType: 'survey',
+        value: [{
+          answers: {},
+          choice: 'WHISTLE',
+          filters: {}
+        }]
+      })
+      surveyAnnotationInProgress.setChoiceInProgress(true)
+    })
+
+    it('should be incomplete', function () {
+      expect(surveyAnnotationInProgress.isComplete).to.be.false()
+    })
+
+    it('should update _choiceInProgress to false when action inProgress is called with false', function () {
+      surveyAnnotationInProgress.setChoiceInProgress(false)
```

Comment:
Small suggestion to make it clear that the action is working as expected:

```suggestion
      expect(surveyAnnotationInProgress._choiceInProgress).to.be.true()
      surveyAnnotationInProgress.setChoiceInProgress(false)
```